src/ReinforcementLearningZoo/src/algorithms/nfsp/nfsp.jl

```diff
@@ -0,0 +1,190 @@
+"""
+Neural Fictitious Self-Play (NFSP) agent implemented in Julia.
+
+See the paper https://arxiv.org/abs/1603.01121 for more details.
+"""
+export NFSPAgent, NFSPAgents
```

Comment:
I don't think users need to directly use `NFSPAgent`, so no need to export this one.